lib/dradis/plugins/saint/importer.rb

```diff
@@ -0,0 +1,97 @@
+module Dradis::Plugins::Saint
+  class Importer < Dradis::Plugins::Upload::Importer
+    def import(params={})
+      @issues = {}
+      @hosts = {}
+      file_content = File.read(params[:file])
+
+      logger.info {'Parsing SAINT output file...'}
+      doc = Nokogiri::XML( file_content )
+      logger.info{'Done.'}
+
+      doc.xpath('/report').each do |xml_report|
+        logger.info {'Processing report...'}
+
+        # Process <vulnerability> tags
+        xml_report.xpath('./details/vulnerability').each do |vuln|
+          process_vuln_issue(vuln)
+        end
+
+        # Process <host> tags
+        xml_report.xpath('./overview/hosts/host').each do |host|
+          process_host_item(host)
+        end
+
+        # Process <vulnerabilities> tag
+        xml_report.xpath('./overview/vulnerabilities/host_info').each do |xml_host_info|
+          host_name = xml_host_info.xpath('./hostname').first.text
+          xml_host_info.xpath('./vulnerability').each do |evidence|
+            process_evidence(evidence, host_name)
+          end
+        end
+
+        logger.info {'Report processed...'}
+      end
+
+      true
+    end
+
+    private
+    def process_vuln_issue(xml_vuln)
+      # Create Dradis Issue
+      logger.info{ "\t\t => Creating new issue..." }
+      plugin_id = Digest::SHA1.hexdigest(xml_vuln.xpath('./description').first.text) 
```

Comment:
if there is no way to uniquely identify each issue, is there a cheaper hashing function we can use?